test/test.go

```diff
@@ -146,6 +146,15 @@ func All(t *testing.T, kind string, config stow.Config) {
 	is.Equal(readItemContents(is, item1), "item one")
 	is.NoErr(acceptableTime(t, is, items[0], item1))
 
+	if ir, ok := item1.(stow.ItemRanger); ok {
```

Comment:
Does `item1` have the `ItemRanger` behavior to test here?

Reply:
It might, if it's an Azure location being tested.

Reply:
s/might/does/